src/cca.jl

```diff
@@ -336,3 +340,113 @@ function fit(::Type{CCA}, X::AbstractMatrix{T}, Y::AbstractMatrix{T};
 
     return M::CCA
 end
+
+abstract type MultivariateTest <: HypothesisTest end
+
+struct WilksLambda <: MultivariateTest
+    stat::Float64
+    fstat::Float64
+    df1::Float64
```

Comment:
Can non-integer degrees of freedom actually arise in practice for these tests?

Reply:
Yes, df2 is generally not an integer.